tripy/examples/resent50/resnet50.ipynb

```diff
@@ -0,0 +1,529 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# ResNet50 Image Classificiation with `tripy` - An Implementation and Demo\n",
+    "\n",
+    "This notebook aims to demonstrate the implementation of a ResNet50 model using `tripy`. We'll explore the architecture, \n",
+    "load pretrained weights, and run predictions on a sample dataset, showcasing how `tripy` can be used effectively for inference of ResNet50 based image classification model.\n",
+    "\n",
+    "### Objectives:\n",
+    "1. Implement ResNet50 using `tripy` and load pretrained weights.\n",
+    "2. Run predictions on sample images and visualize results.\n"
```

Comment:
I think we can drop the objectives section since we already explained what we're going to do above?
In general, let's try to keep things brief. Assume short attention spans :slightly_smiling_face:

Reply:
removed